eoscompanion/service.py

```diff
@@ -20,12 +20,47 @@
 
 from gi.repository import (
     EosCompanionAppService,
+    Gio,
     GObject
 )
 
 from .server import create_companion_app_webserver
 
 
+def yield_monitors_over_changed_file_in_paths(paths, callback):
+    '''Yield Gio.FileMonitor objects over each .changed file in :paths:.'''
```

Comment:
Flatpak has a file monitor. Can't that be used? https://lazka.github.io/pgi-docs/Flatpak-1.0/classes/Installation.html#Flatpak.Installation.create_monitor

Reply:
Skip that, not possible as we are running inside a Flatpak ourselves.